crates/wysiwyg/src/composer_model/replace_text.rs

```diff
@@ -176,6 +188,54 @@ where
         self.create_update_replace_all()
     }
 
+    fn set_new_text_in_sibling_node(
+        &mut self,
+        node_handle: DomHandle,
+        new_text: S,
+    ) {
+        if let Some(sibling_text_node) =
+            self.first_sibling_text_node_mut(&node_handle)
+        {
+            let mut data = sibling_text_node.data().to_owned();
+            data.insert(0, &new_text);
+            sibling_text_node.set_data(data);
+        } else if !new_text.is_empty() {
+            let new_child = DomNode::new_text(new_text);
+            let parent = self.state.dom.parent_mut(&node_handle);
+            let index = node_handle.index_in_parent() + 1;
+            parent.insert_child(index, new_child);
+        }
+    }
+
+    fn first_sibling_text_node_mut(
```

Comment:
`fn find_next_text_node_mut`?

Reply:
maybe `find_next_sibling_text_node_mut` I feel that not specifying sibling, makes that next mistakable for maybe the next node in the hierarchy or so on...

Reply:
Yes, that makes more sense.